x/cert/client/cli/query.go

```diff
@@ -46,14 +48,18 @@ func cmdGetCertificates() *cobra.Command {
 			}
 
 			params := &types.QueryCertificatesRequest{
-				Filter: types.CertificateFilter{
-					Owner:  "",
-					Serial: cmd.Flag("serial").Value.String(),
-					State:  "",
-				},
 				Pagination: pageReq,
 			}
 
+			if value := cmd.Flag("serial").Value.String(); value != "" {
+				val, valid := new(big.Int).SetString(value, 10)
+				if !valid {
+					return errors.Errorf("invalid value in serial flag. expected integer")
```

Comment:
Since this is the same error message each time, it makes more sense to use an assignment like `var errInvalidCertificate = errors.new("invalid value in serial flag. expected integer")`